### DISCUSSION TITLE
Welcome to cppstories-discussions Discussions!

### DISCUSSION THREAD

**Discussion by fenbf** [fenbf/cppstories-discussions#13] [General]
## 👋 Welcome!

`cppstories-discussions` is a forum around [cppstories.com](https:www.cppstories.com) - a place where you can learn about the latest C++ features, best practices and news from the C++ world.

**Currently this forum is in the alpha stage**.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by houba80** [comment]
hello everyone 
i new in programming i learned a few python and learned full stack web development but i still not have the experience so i am tring to 
get it from this community ..
now i have a problem ,, i have an old program i think it written on Turbo Pascal it financial software and request for login password 
i need to get this password because the user of this software is left the company long time ago,, anyone can help me
thanks for advance!
